helpdesk/email.py

```diff
@@ -84,16 +90,24 @@ def process_email(quiet=False):
             logger.addHandler(log_file_handler)
         else:
             log_file_handler = None
-
-        try:
             if not q.email_box_last_check:
                 q.email_box_last_check = timezone.now() - timedelta(minutes=30)
-
+        try:
             queue_time_delta = timedelta(minutes=q.email_box_interval or 0)
             if (q.email_box_last_check + queue_time_delta) < timezone.now():
                 process_queue(q, logger=logger)
                 q.email_box_last_check = timezone.now()
                 q.save()
+                log_msg: str = f"Queue successfully processed: {q.slug}"
+                logger.info(log_msg)
+                if debug_to_stdout:
+                    print(log_msg)
+        except Exception as e:
+            logger.error("Queue processing failed: {q.slug} -- {e}", exc_info=True)
```

Comment:
Does this work without the `f` at the beginning of the string?

Reply:
Good catch. I have fixed it and added unit tests